MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+
+[SCK values.yaml snippet]
+```
+logs:
+  kube-audit:
+    from:
+      file:
+        path: /var/log/kube-apiserver-audit.log
+    timestampExtraction:
+      format: "%Y-%m-%dT%H:%M:%SZ"
+    sourcetype: kube:apiserver-audit
+```    
+
+[SCK-OTEL values.yaml snippet]
+```
+extraFileLogs:
+ filelog/kube-audit:
+   include: [/var/log/kube-apiserver-audit.log]
+   start_at: beginning
+   include_file_path: true
+   include_file_name: false
+   resource:
+     service.name: /var/log/kube-apiserver-audit.log
+     host.name: 'EXPR(env("K8S_NODE_NAME"))'
+     com.splunk.sourcetype: kube:apiserver-audit
+```     
+
+You must use the same keyword ("kube-audit" in the above example) for continuing to read from the translated checkpoint data.
+
+### Translating custom configs from SCK to Splunk OpenTelemetry Collector for Kubernetes for metrics
+#### Limiting/Increasing Application Resources
+If you are using the resources option in SCK to limit/increase the CPU and memory usage in SCK, you can do the same using the resources option in "clusterDataCollector" section in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Scheduling configs
+If you are using any pod scheduling operations such as nodeSelector, affinity and tolerations in SCK, you can use the nodeSelector, affinity and tolerations options in Splunk OpenTelemetry Collector for Kubernetes.
+
+#### Pod Security Context
+If you are using the podSecurityContext option to set a pod security policy, you can use the podSecurityContext option in SCK-OTEL.
+
+#### Custom Annotations
+If you are using the customMetadataAnnotations option to set a custom annotation fields in your logs in SCK, you can use the "annotations" and "podAnnotations" option in Splunk OpenTelemetry Collector for Kubernetes.
+
+## Step 2: Delete SCK Deployment (Optional)
+If you want to delete your SCK deployment:
+
+* Get your current SCK deployment name by running this command:
+
+  * helm ls
+
+* **If you want to delete only logs and metrics**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete only logs**
+
+  * Update the values.yaml file used to deploy SCK to disable objects and run the following command:
+
+  * helm upgrade local-k8s -f your-values-file.yaml splunk/splunk-connect-for-kubernetes
+
+* **If you want to delete your entire SCK deployment (logs, objects and metrics), run the following command**
+
+  * helm delete local-k8s
+
+## Step 3: Installing Splunk OpenTelemetry Collector for Kubernetes
+
+* Follow the README [here](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/README.md).
+
+## Step 4: Check Data in Splunk
```

Comment:
"Step 4: Check data in Splunk"

"Check the logs index to see if you are receiving logs from your Kubernetes cluster."

"Check the metrics index to see if you are receiving logs from your Kubernetes cluster."